specification/metadata/focusversion.md

```diff
@@ -0,0 +1,17 @@
+# FOCUS Version
+
+The declared FOCUS version of the FOCUS data within the schema.
+
+The FOCUSVersion MUST be provided in the metadata. FOCUSVersion MUST be of type String and MUST NOT contain null values. FOCUSVersion MUST match one of the published versions of the FOCUS specification. FOCUSVersion MUST match the version of the FOCUS specification that the FOCUS dataset conforms to.
+
+## Metadata ID
+
+FOCUSVersion
```

Comment:
FocusVersion?

Same rationale as for SkuId, SkuPriceId

Reply:
+1

Reply:
Updated